README.md

```diff
@@ -1,3 +1,25 @@
+## Requirements
+
+* scrapy>=2.5.1
+* zyte-api>=0.1.2
+* twisted>=21.7.0
```

Comment:
May I ask why these explicit requirements are needed?

For example:

- `zyte-api>=0.1.2` is already set here: https://github.com/scrapy-plugins/scrapy-zyte-api/blob/main/setup.py#L13
- min version for `twisted` on `scrapy==2.5.1` is `17.9.0` as set here: https://github.com/scrapy/scrapy/blob/2.5/setup.py#L22

Moreover, was there any bugs when `scrapy` *2.3* or *2.4* is used?

Reply:
>Moreover, was there any bugs when scrapy 2.3 or 2.4 is used?

I am not sure about the bugs since I have not tested them in those versions. I mentioned these in the requirement section as part of the documentation. So it gives more information on what are packages or modules being used for this scrapy-plugin. 

>May I ask why these explicit requirements are needed?

I took references from other scrapy-plugins for creating the documentation such as https://github.com/scrapy-plugins/scrapy-crawlera-fetch

Let me know if you want me to remove this part I will remove it.

Reply:
For this case, I believe we can remove these parts as performing a `pip install` would automatically install them. There's no need for the user to have a special method for installing them separately.

Reply:
Yes makes more sense now.